front/pages/api/w/[wId]/data_sources/[name]/managed/config/[key]/index.ts

```diff
@@ -10,18 +10,18 @@ import { Authenticator, getSession } from "@app/lib/auth";
 import logger from "@app/logger/logger";
 import { apiError, withLogging } from "@app/logger/withlogging";
 
-export const PostBotEnabledRequestBodySchema = t.type({
-  botEnabled: t.boolean,
+export const PostManagedDataSourceConfigRequestBodySchema = t.type({
+  configValue: t.string,
```

Comment:
Why string over boolean?

Reply:
A config is a string as per ConnectorsAPI :+1: